oaknational/Oak-Web-Application
# PUPIL-249 feat(oak-layout): add oak layout to quiz

Add quiz layout to quiz view

## How to test

1. Go to https://deploy-preview-2207--oak-web-application.netlify.thenational.academy/pupils/programmes/english-secondary-ks4-aqa/units/macbeth-and-masculinity-the-struggle-for-power/lessons/duncan-as-a-father-figure
2. Click on quizzes
3. You should see quiz layout

## Screenshots

How it used to look (delete if n/a):
{screenshots}

How it should now look:
{screenshots}

## Checklist

- [x]  Use oakLessonLayout
- [x]  MCQs display ‘select x answers’
- [x]  Question count
- [x]  Back button and header
- [x]  Feedback in bottom nav
- [x]  Different colour backgrounds based on starter or exit quiz
- [x]  mobile, tablet browser views
- [x]  final button is ‘continue lesson’ instead of ‘next question’
